src/ai/backend/manager/models/error_logs.py

```diff
@@ -0,0 +1,36 @@
+import enum
+
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+from .base import metadata, IDColumn, GUID
+__all__ = [
+    'error_logs',
+    'LogSeverity'
+]
+
+
+class LogSeverity(str, enum.Enum):
+    CRITICAL = 'critical'
+    ERROR = 'error'
+    WARNING = 'warning'
```

Comment:
For future, let's add INFO, DEBUG levels as well.
(Don't forget to update the migration!)